README.md

```diff
@@ -231,7 +227,6 @@ and CITGM team members listed below.
 * [@BridgeAR](https://github.com/BridgeAR) - Ruben Bridgewater
 * [@bzoz](https://github.com/bzoz) - Bartosz Sosnowski
 * [@gdams](https://github.com/gdams) - George Adams
-* [@MylesBorins](https://github.com/MylesBorins) - Myles Borins
```

Comment:
Was this supposed to be the entry from line 214 in the diff?

Reply:
Yes, whoops! I'll fix it.